src/components/no-match.js

```diff
@@ -8,14 +8,39 @@ import {
   AppFooter,
   AppMain,
   ErrorDart,
-  ErrorRabbit
+  ErrorRabbit,
+  urlWhoAmI,
 } from 'formula_one'
 
-import main from '../css/app.css'
```

Comment:
Why these CSS files are removed? It must not be working without this. Did you test it running?